pkg/resource/notebook_instance/hooks.go

```diff
@@ -37,8 +41,8 @@ func (rm *resourceManager) requeueUntilCanModify(
 	ctx context.Context,
 	r *resource,
 ) error {
-	latestStatus := r.ko.Status.NotebookInstanceStatus
-	return svccommon.RequeueIfModifying(latestStatus, &resourceName, &modifyingStatuses)
+	notebookStatus := r.ko.Status.NotebookInstanceStatus
```

Comment:
why the change here from latestStatus to notebookStatus ?

Reply:
I think I changed it in response to another PR comment.